lang/java/reef-examples/pom.xml

```diff
@@ -117,6 +117,11 @@ under the License.
             <artifactId>reef-runtime-multi</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.reef</groupId>
+            <artifactId>reef-runtime-azbatch</artifactId>
+            <version>0.17.0-SNAPSHOT</version>
```

Comment:
I believe there is already a dependency declared for reef-runtime-azbatch on line 44. Is this necessary? If it is, let's change version to ${project.version}

Reply:
My intelliJ refused to compile it yesterday without this. I tried cleaning my project and it works now. I will remove this.